### PR TITLE
feat: interaction's reactions

### PR DESCRIPTION
**Pull request by JohnBerd**
- reaction counter on a message
- button to add a reaction if a reaction is already on a message
- reactions of the current user are colored
- a long press on a reaction displays the list of user having the same reaction on a message

### before
testgifbefore

### after
testgif

### CODE REVIEW

**Review by clegirar — CHANGES_REQUESTED** [review]
Except the memoized functions and the log, the code seems good (i don't test the PR, @zxxma did it), so LGTM after changes 👍
Maybe just change the PR's name :)

File: js/packages/components/shared-components/AnimatedNumber.tsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+			Animated.timing(animation, {
+				toValue: -1 * (numberHeight * animateToNumbersArr[index]),
+				duration: animationDuration || 1400,
+				easing: easing || EasingNode.elastic(1.2),
+			}).start()
+		})
+	}, [number, animateToNumbersArr, animationDuration, animations, easing, numberHeight])
+
+	const getTranslateY = (index: number) => {
```
Comment (clegirar) [inline js/packages/components/shared-components/AnimatedNumber.tsx:57]: Sorry i forgot this function, same for the memoized hook, i think we have to use a `useCallback` here to

Comment (JohnBerd) [reply js/packages/components/shared-components/AnimatedNumber.tsx:57]: Thank you for the constructive review <3

**Review by clegirar — APPROVED** [review]
LGTM 👍

**Pull request merged**